web/session.go

```diff
@@ -125,6 +125,35 @@ func loginOK(r *rest.Request) bool {
 	return true
 }
 
+func loginWithTokenOK(r *rest.Request, token string) bool {
+	restToken, err := auth.ParseRestToken(token)
+	//validToken, err := auth.ValidateRestToken(r.Request, token)
```

Comment:
This should maybe be uncommented? Something needs to validate that `token.RequestHash()` matches `GetRequestHash(r.Request)`